ProcessesApi.Tests/V1/E2ETests/Stories/UpdateProcessByIdTests.cs

```diff
@@ -0,0 +1,91 @@
+using Hackney.Core.Testing.DynamoDb;
+using Hackney.Core.Testing.Sns;
+using ProcessesApi.Tests.V1.E2E.Fixtures;
+using ProcessesApi.Tests.V1.E2ETests.Steps;
+using ProcessesApi.V1.Domain;
+using System;
+using TestStack.BDDfy;
+using Xunit;
+
+namespace ProcessesApi.Tests.V1.E2E.Stories
+{
+    [Story(
+       AsA = "Internal Hackney user (such as a Housing Officer or Area Housing Manager)",
+       IWant = "The system to automatically check a tenant and an applicants eligibility for a Sole to Joint application",
+       SoThat = "I can more quickly determine if I should continue with the application")]
```

Comment:
Is this ticket about the automated eligibility checks?

Maybe it should be something like:
  I want to update a process without updating the state
  So that I can update case details without changing the stage it is in(?)